orthw

```diff
@@ -143,13 +143,13 @@ usage() {
   echo "  analyze <source-code-dir>"
   echo "  analyze-in-docker <source-code-dir>"
   echo "  create-analyzer-result <package-ids-file>"
-  echo "  delete-scan-results <package-id>"
+  echo "  delete-package-provenances <package-id>"
```

Comment:
Commit message typos:
- "can-results`"
- "an `provenances_scan_results`" -> "and `provenances_scan_results`"